Libraries/Microsoft.Teams.Apps.Testing/Events/App.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.Teams.Apps.Events;
+using Microsoft.Teams.Apps.Testing.Plugins;
+
+namespace Microsoft.Teams.Apps.Testing.Events;
+
+public class TestMessageEvent : Event
+{
+    public required string Message { get; set; }
+}
+
+public static partial class AppExtensions
+{
+    public static App OnTestMessage(this App app, Action<TestPlugin, TestMessageEvent> handler)
+    {
+        return app.OnEvent("test.message", (plugin, @event) =>
```

Comment:
is this "test." enforced? Or convention

Reply:
its namespaced, so any events the test plugin emits will be prefixed with `test.`.